ts/src/base/Exchange.ts

```diff
@@ -3234,6 +3234,32 @@ export default class Exchange {
         return ohlcvs;
     }
 
+    async buildOHLCVCUsingWatchTrades (symbol, timeframe: string = '1m', since: number = 0, limit: number = 2147483647, callbackFunction = undefined, params = {}) {
+        let collectedTrades = [];
+        const collectedBars = [];
+        while (true) {
+            const wsTrades = await this.watchTrades (symbol, since, limit, params);
```

Comment:
I don't think we should handle this call by ourselves, it might throw several different errors (connection, timeouts, etc). why not move this to an example? making clear that the watchTrades call is controlled by the user

Reply:
ahh, exception handling is a thing there. oky, right.